copper_sdk/copper.py

```diff
@@ -42,6 +50,20 @@ def put(self, endpoint, opts):
     def delete(self, endpoint):
         return self.api_call('delete', endpoint)
 
+    @retry(Exception, tries =RETRY_COUNT, delay=RETRY_INTERVAL)
+    def get_response(self, method, endpoint, optsJson):
```

Comment:
Did we fetch merge the latest copper sdk ?

Reply:
@makeshmakesh I think we can merge this for now 
We should fetch merge latest since there was some updates happened on origin